### PR TITLE
*: cut v1.7.2 release

### PR DESCRIPTION
**Pull request by LiliC**
cc @brancz @tariq1890

### CODE REVIEW

**Comment by brancz** [comment]
lgtm, giving last call to @tariq1890 

@loburm @serathius please stand by to publish the images to gcr.io :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## v1.7.2 / 2019-08-05
+
+* [CHANGE] Revert "add kube_*_annotations metrics for all objects" #859
+* [CHANGE] Remove kube_namespace_annotations metric #859
```
Comment (tariq1890) [inline CHANGELOG.md]: Why the Italics here?

Comment (tariq1890) [reply CHANGELOG.md]: or rather the lack of italics in the previous line :).

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## v1.7.2 / 2019-08-05
+
+* [CHANGE] Revert "add kube_*_annotations metrics for all objects" #859
```
Comment (tariq1890) [inline CHANGELOG.md]: Is it me or does this text look inconsistent?

File: CHANGELOG.md (hunk 1 above)
Comment (LiliC) [reply CHANGELOG.md]: Aha I think its because of the above commit msg, the `_*_`

File: CHANGELOG.md (hunk 2 above)
Comment (LiliC) [reply CHANGELOG.md]: Fixed, PTAL thanks!

File: README.md [hunk 3]
```diff
@@ -62,6 +62,7 @@ At most, 5 kube-state-metrics and 5 [kubernetes releases](https://github.com/kub
 | **v1.5.0**         |  v8.0.0    |         ✓           |         ✓           |         -           |         -           |          -           |
 | **v1.6.0**         |  v11.0.0   |         ✓           |         ✓           |         ✓           |         ✓           |          -           |
 | **v1.7.1**         |  v12.0.0   |         ✓           |         ✓           |         ✓           |         ✓           |          ✓           |
```
Comment (tariq1890) [inline README.md]: We should this line IMO.

**Comment by brancz** [comment]
Image published at `quay.io/coreos/kube-state-metrics:v1.7.2` and `quay.io/coreos/kube-state-metrics:latest`.

**Pull request merged**